ml-package-versions.yml

```diff
@@ -152,3 +152,22 @@ lightgbm:
     requirements: ["scikit-learn", "matplotlib"]
     run: |
       pytest tests/lightgbm/test_lightgbm_autolog.py --large
+
+
+gluon:
+  package_info:
+    pip_release: "mxnet"
+    install_dev: |
+      pip install --pre mxnet -f https://dist.mxnet.io/python/cpu
+
+  models:
+    minimum: "1.5.1"
+    maximum: "1.7.0.post1"
```

Comment:
`1.7.0` doesn't exist on PyPI. I guess it was removed after `1.7.0.post1` was released:

https://pypi.org/project/mxnet/#history